gorm-rest/src/main/groovy/yakworks/rest/gorm/controller/BulkControllerSupport.groovy

```diff
@@ -141,4 +150,20 @@ class BulkControllerSupport<D> {
         RepoLookup.findRepo(getEntityClass())
     }
 
+    /**
+     * Special handler for bulk operations, so that we can log/highight every bulk error we send.
+     * Its here, because we cant have more thn one exception handler for "Exception" in controller
+     */
+    Problem handleBulkOperationException(HttpServletRequest req, Exception e) {
+        String message = "requestURI=[${req.requestURI}], method=[${req.method}], queryString=[${req.queryString}]"
+
+        Problem apiError = problemHandler.handleException(getEntityClass(), e)
+
+        if (apiError.status.code == 500) {
+            log.error("⚠️ Bulk operation exception ⚠️ \n $message", apiError.cause)
```

Comment:
look at what we do in the params. log it the same way (as a warning with all the request info) so we can turn it off. please re-read the initial comment above and try to understand what we are doing here. we are trying to figure out what they are doing to call us so we can diagnose and send it back to them better.

Reply:
we are also trying not to pollute the logs with huge stacktraces for items that are not our deal so doubling up on it will not help but make it more polluted.